apis/r/R/SOMADataFrame.R

```diff
@@ -19,14 +19,21 @@ SOMADataFrame <- R6::R6Class(
     #' @param index_column_names A vector of column names to use as user-defined
     #' index columns.  All named columns must exist in the schema, and at least
     #' one index column name is required.
-    #' @param domain An optional list of 2-element vectors specifying the domain of each index
-    #' column. Each vector should be a pair consisting of the minimum and maximum values storable in
-    #' the index column. For example, if there is a single int64-valued index column, then `domain`
-    #' might be `c(100, 200)` to indicate that values between 100 and 200, inclusive, can be stored
-    #' in that column.  If provided, this list must have the same length as `index_column_names`,
-    #' and the index-column domain will be as specified.  If omitted entirely, or if `NULL` in a given
-    #' dimension, the corresponding index-column domain will use the minimum and maximum possible
-    #' values for the column's datatype.  This makes a `DataFrame` growable.
+    #' @param domain An optional sequence of tuples specifying the domain of each
+    #' index column. Each tuple must be a pair consisting of the minimum and
```

Comment:
`tuple` is Python-specific vocabulary; can we change this to `vector` for the R docs since tuples do not exist in R?
```suggestion
    #' @param domain An optional sequence of vectors specifying the domain of each
    #' index column. Each vector must be a pair consisting of the minimum and
```

Reply:
Good catch, thanks @mojaveazure !

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/3396/commits/0a89bb6ed12d144f8ef8f9f4bf8dd659ef1c5f3e

Ready for round two, thank you!